### PR TITLE
Jannie and Miner

### PR DESCRIPTION
**Pull request by Lamasmaster**
## About The Pull Request
<details>
<summary>
	About The Pull Request
</summary>
<hr>

Makes janitor bit more like janitor (why tf is he using jackboots instead of galoshes ?) removing slowdown from them. Changes around preps a little bit for janitor and miner
jannie1
jannie2

	
<hr>
</details>

## Changelog
:cl:
tweak: Miner and Janitor
/:cl:

### CODE REVIEW

File: code/modules/mining/mine_items.dm [hunk 1]
```diff
@@ -18,23 +18,17 @@
...
-	new /obj/item/tool/pickaxe(src)
 	new /obj/item/tool/pickaxe/drill(src)
+	new /obj/item/tool_upgrade/augment/fuel_tank(src)
 	new /obj/item/device/t_scanner(src)
 	new /obj/item/gun/projectile/boltgun/flare_gun(src)
 	new /obj/item/ammo_casing/flare(src)
 	new /obj/item/device/ore_sonar(src)
 	new /obj/item/ammo_magazine/speed_loader_shotgun/empty(src)
-	new /obj/item/clothing/accessory/job/cape/mining(src) // ROCK AND STONE, BRUDDAH!
-	new /obj/item/gun/projectile/shotgun/pump(src)
```
Comment (Trilbyspaceclone) [inline code/modules/mining/mine_items.dm:34]: ammo but no gun?

File: code/modules/mining/mine_items.dm [hunk 2]
```diff
@@ -18,23 +18,17 @@
 	new /obj/item/cell/medium(src)
 	new /obj/item/cell/medium(src)
 	new /obj/item/cell/small(src)
-	new /obj/item/tool_upgrade/augment/fuel_tank(src)
 	new /obj/item/device/scanner/gas(src)
 	new /obj/item/storage/bag/ore(src)
-	new /obj/item/device/lighting/toggleable/flashlight/heavy(src)
-	new /obj/item/tool/shovel(src)
```
Comment (Trilbyspaceclone) [inline code/modules/mining/mine_items.dm:25]: We kinda need a shovel for digging sand that dosnt take 40000 fuel

File: code/game/objects/structures/crates_lockers/closets/job_closets.dm [hunk 3]
```diff
@@ -72,18 +72,14 @@
...
-	new /obj/item/caution(src)
-	new /obj/item/caution(src)
 	new /obj/item/device/lightreplacer(src)
 	new /obj/item/storage/bag/trash(src)
 	new /obj/item/clothing/shoes/galoshes(src)
 	new /obj/item/mop(src)
+	new /obj/item/tool/shovel(src)
+	new /obj/item/reagent_containers/spray/cleaner(src)
 	new /obj/item/soap/deluxe(src)
-	new /obj/item/newspaper_stack(src)
```
Comment (Trilbyspaceclone) [inline code/game/objects/structures/crates_lockers/closets/job_closets.dm:84]: This is needed for the job

**Comment by ThePainkiller** [comment]
Galoshes had slowdown because they've been historically noslip shoes I guess

**Pull request merged**